day2/exercise3/solution.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"sync"
+	"time"
+)
+
+var (
+	state     sync.Mutex
+	balance int
```

Comment:
Lint: run `go fmt path/to/your/package`
```suggestion
	state   sync.Mutex
	balance int
```